src/components/InstancesList/InstancesList.js

```diff
@@ -548,7 +548,7 @@ class InstancesList extends React.Component {
     }
   }
 
-  toggleNewFastAddModal = ({ instanceRecord }) => {
+  toggleNewFastAddModal = ({ instanceRecord } = {}) => {
```

Comment:
If I'm not mistaken on line 739 `buildOnClickHandler` calls this function with arg `this.context.sendCallout` which is a function, not an object with `instanceRecord` prop. Looks like something is not correct here

Reply:
Nice catch @mariia-aloshyna, I added a separate function to handle closing of the modal. It will always receive the same type of argument